modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -5,49 +5,61 @@
 label:AuraDB-Enterprise[]
 label:AuraDS-Enterprise[]
 
-Aura Enterprise supports Single Sign-On (SSO) at both the Console level and for accessing Workspace, Bloom and Browser clients directly at the instance level.
+== SSO levels
+
+Organization admins can configure organization level SSO (org SSO) and tenant level SSO (tenant SSO). 
+SSO is simply a log-in method and all access/roles/permissions are dictated by RBAC.
+
+* *Org SSO:* Just because you can login via org SSO doesn't mean you have access to the org. 
+You still have to be invited and the invitee will grant you a role when you are invited and that dictates if you can edit settings or view/manage instances. 
+Org SSO allows org admins to restrict how users login when they are trying to access the org. 
+Everything else around access is managed via roles (RBAC) and invites.
 
-[NOTE]
-====
-Accessing Aura with SSO requires:
- 
-* Authorization Code Flow with PKCE.
-* A publicly accessible Identity Provider (IdP) server.
-====
 
-== Console SSO
+* *Tenant level SSO:*  Impacts new database instances created within that tenant. It ensures users logging in with SSO have access to the database instances within the tenant. It depends on RBAC if the user can access and view or modify data within the database instances themselves. For this level, the role mapping may be used to grant users different levels of access based on a role in their IdP (it *does not* give access to edit the tenant settings e.g. edit the tenant name / network access, or edit the instance settings e.g. rename an instance / pause and resume).
 
-Console SSO allows you to log in to the Aura Console using company IdP credentials and grants link:{neo4j-docs-base-uri}/cypher-manual/current/administration/access-control/built-in-roles#access-control-built-in-roles-public[Public Access privileges] to all instances in the tenant.
+== Login methods
 
-The following OpenID Connect (OIDC) certified Identity Providers (IdPs) are currently supported for Console-level Authentication:
+Login methods are different for each SSO level.
+Administrators can configure a combination of one or more of the login methods.
 
+*Organization level SSO supports:*
+
+* Email/password
+* Okta
 * Microsoft Entra ID
+* Google SSO
+
+An organization's administrator can add Aura as a login from a tile in an organization's Apps Dashboard.
+
+*Instance level SSO supports:*
```

Comment:
This level has not been mentioned before. Above you list two levels, org and tenant. Is instance-level different from tenant?